### PR TITLE
Simplified StyleCI config

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: .styleci.yml [hunk 1]
```diff
@@ -3,13 +3,4 @@ preset: recommended
...
-    exclude:
-        - "tests"
-    name:
-        - "*.php"
-    path:
-        - "src"
-    not-path:
-        - "tests"
-    depth:
-        - "< 3"
```
Comment (GrahamCampbell) [inline .styleci.yml:15]: Not quite sure why you set that?

Comment (kisphp) [inline .styleci.yml:15]: Not sure either. Just copied the example and played with it to see how it works. I'll bring more changes to it while I'll read the documentation.

thanks for mention that.

Comment (GrahamCampbell) [inline .styleci.yml:15]: I did add a note to the docs saying don't copy the example config last week.

Comment (GrahamCampbell) [inline .styleci.yml:15]: > Please note that we do not recommend actually copy/pasting this example file for use as it's pretty useless for most applications, but it's useful to demonstrate all the available options.

Comment (GrahamCampbell) [inline .styleci.yml:15]: https://github.com/StyleCI/Config/blob/master/src/ConfigFactory.php#L74 let's you see the config defaults.

Comment (GrahamCampbell) [inline .styleci.yml:15]: and https://github.com/StyleCI/Config/blob/master/src/ConfigFactory.php#L40 too

Comment (kisphp) [inline .styleci.yml:15]: yes, I saw that, but as I said, I need to get into all features in the doc and complete the configuration file. This was done quickly with later pending work.

By the way, will you provide integration for bitbucket too ?

Comment (GrahamCampbell) [inline .styleci.yml:15]: No, we only support GitHub.

**Pull request merged**